src/home/devrel-tracker.ts

```diff
@@ -13,12 +13,12 @@ export function initiateDevRelTracking() {
   }
 }
 
-export async function trackDevRelReferral(devGitHubId: number) {
+export async function trackDevRelReferral() {
   const devRelCode = localStorage.getItem("devRel");
 
   // key: user_id (devGitHubId), value: referral_id (devRelCode)
   if (devRelCode && devRelCode != "done") {
-    const url = `${WORKER_URL}/tracker?key=${encodeURIComponent(devGitHubId)}&value=${encodeURIComponent(devRelCode)}`;
+    const url = `${WORKER_URL}/tracker`;
```

Comment:
you don't need worker URL because it is the same as the page URL so you can just use `/tracker`